src/main/java/by/exadel/internship/entity/Form.java

```diff
@@ -65,6 +68,9 @@
 
     @Column(name = "fm_primary_skill")
     private String primarySkill;
+    
+    @Column(name = "fm_deleted")
+    private Boolean deleted;
```

Comment:
It should have default false value, so no reason to use Boolean. Correct in other places as well

Reply:
Done